### DISCUSSION TITLE
Add env variable to make next/image unoptimised

### DISCUSSION THREAD

**Discussion by maksimf** [vercel/next.js#38960] [Help] [answered]
### Describe the feature you'd like to request

Sometimes developer might want to make all next/image components `unoptimized`, e.g. when running next.js code outside of the next.js server (storybook, ladle, etc). 

### Describe the solution you'd like

It would be really convinient if nextjs would expose an environment variable, which would make all images unoptimized

### Describe alternatives you've considered

For storybook there is [this solution](https://github.com/vercel/next.js/issues/18393#issuecomment-909636489), which is basically monkey-pathing all next/image components

**Comment by balazsorban44** [comment, marked as answer]
You should be able to set this in your `next.config.js` that delegates to all of your images. Please see https://nextjs.org/docs/api-reference/next/image#unoptimized
